apps/angular/src/app/features/anime/anime-details/anime-details.component.ts

```diff
@@ -38,6 +42,24 @@ export class AnimeDetailsComponent {
 		);
 	}
 
+	/**
+	 * Get genre names.
+	 * @param animeDetails - Anime details.
+	 */
+	protected getGenreNames(animeDetails: AnimeDetails): readonly string[] {
```

Comment:
```suggestion
	protected getGenreNames(animeDetails: AnimeDetails): string[] {
```

There is no need to describe the return type as readonly. Usually readonly is used as a helper when writing pure functions. For example, we specify readonly for an array in parameters so that we can't change an incoming parameters inside function. The same is true for an object.